.github/workflows/on-pr.yaml

```diff
@@ -46,7 +54,7 @@ jobs:
 
         # Just testing the AWS Creds 
         aws --debug --profile rarecamp_dev cloudformation list-stacks
-        AWS_PROFILE=rarecamp_dev NODE_ENV=production ALARMS_NOTIFICATION_EMAIL=alert@example.com SERVERLESS_SERVICE_SUFFIX=-${{ github.head_ref }} npm run deploy:dev
+        AWS_PROFILE=rarecamp_dev NODE_ENV=production ALARMS_NOTIFICATION_EMAIL=alert@example.com SERVERLESS_SERVICE_SUFFIX=-${{ github.event.number }} npm run deploy:dev
```

Comment:
Using event.number (the PR ID) instead of the branch name because long branch names cause Serverless Framework/CloudFormation long logical resource id errors.